sphinx_markdown_builder/translator.py

```diff
@@ -327,6 +328,9 @@ def visit_image(self, node):
         # It will be handled by the visit/depart handlers of the paragraph.
         self.add(f"![{alt}]({uri})")
 
+    def visit_figure(self, node):
+        return None
+
```

Comment:
Instead of an empty handler, I use a dict of predefined elements at the top of this file: `PREDEFINED_ELEMENTS`.
Simply add there the following line: `figure=None`.
Then, remove this method.

Reply:
Ohh..I didn't see that,
It is now updated :)